backup_and_restore/application_backup_and_restore/features-and-plugins/oadp-configuring-velero-plugins.adoc

```diff
@@ -0,0 +1,73 @@
+:_content-type: ASSEMBLY
+[id="oadp-velero-plugins"]
+= OADP Velero plugins
+include::_attributes/common-attributes.adoc[]
+:context: oadp-features-plugins
+
+toc::[]
+
+You can configure two types of plugins when you install Velero:
+
+* Default cloud provider plugins
+* Custom plugins
+
+Both types of plugin are optional, but most users configure at least one cloud provider plugin.
+
+== Default Velero cloud provider plugins
+
+You can install any of the following default Velero cloud provider plugins when you configure the `oadp_v1alpha1_dpa.yaml` file during deployment:
+
+* `aws` (Amazon Web Services)
+* `gcp` (Google Cloud Platform)
+* `azure` (Microsoft Azure)
+* `openshift` (OpenShift Velero plugin)
+* `csi` (Container Storage Interface)
+// removed * `kubevirt` (KubeVirt)
+
+You specify the desired default plugins in the `oadp_v1alpha1_dpa.yaml` file during deployment.
+
+.Example file
+
+The following `oadp_v1alpha1_dpa.yaml` file installs the `openshift`, `aws`, `azure`, and `gcp` plugins:
+
+[source,yaml]
+----
+ apiVersion: oadp.openshift.io/v1alpha1
+ kind: DataProtectionApplication
+ metadata:
+   name: dpa-sample
+ spec:
+   configuration:
+     velero:
+       defaultPlugins:
+       - openshift
+       - aws
+       - azure
+       - gcp
+----
+
+== Custom Velero plugins
```

Comment:
Anchor/ID string needed for all headings [per the docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings).